packages/atomic/src/components/commerce/atomic-commerce-facet/atomic-commerce-facet.ts

```diff
@@ -0,0 +1,429 @@
+    return this.renderValuesContainer(
+      this.facetState.facetSearch.values.map(
+        (value) =>
+          html`${renderFacetValue({
+            props: {
+              ...this.facetValueProps,
+              facetState: 'idle',
+              facetCount: value.count,
+              onExclude: () => this.facet.facetSearch.exclude(value),
+              onSelect: () => this.facet.facetSearch.select(value),
+              facetValue: value.rawValue,
+            },
+          })}`
+      )
+    );
+  }
+
+  private renderValues() {
+    return this.renderValuesContainer(
+      this.facetState.values.map((value, i) => {
+        const shouldFocusOnShowLessAfterInteraction = i === 0;
+        const shouldFocusOnShowMoreAfterInteraction = i === 0;
+
+        return renderFacetValue({
+          props: {
+            ...this.facetValueProps,
+            facetCount: value.numberOfResults,
+            onExclude: () => this.facet.toggleExclude(value),
+            onSelect: () => this.facet.toggleSelect(value),
+            facetValue: value.value,
+            facetState: value.state,
+            setRef: (btn) => {
+              if (shouldFocusOnShowLessAfterInteraction) {
+                this.showLessFocus?.setTarget(btn as HTMLElement);
+              }
+              if (shouldFocusOnShowMoreAfterInteraction) {
+                this.showMoreFocus?.setTarget(btn as HTMLElement);
+              }
+            },
+          },
+        });
+      })
+    );
+  }
+
+  private renderShowMoreLess() {
+    return html`
+      ${renderFacetShowMoreLess({
+        props: {
+          label: this.displayName,
+          i18n: this.bindings.i18n,
+          canShowLessValues: this.facetState.canShowLessValues,
+          canShowMoreValues: this.facetState.canShowMoreValues,
+          onShowLess: () => {
+            this.focusTargets.showLess.focusAfterSearch();
+            this.facet.showLessValues();
+          },
+          onShowMore: () => {
+            this.focusTargets.showMore.focusAfterSearch();
+            this.facet.showMoreValues();
+          },
+        },
+      })}
+    `;
+  }
+
+  private renderMatches() {
+    return renderFacetSearchMatches({
+      props: {
+        i18n: this.bindings.i18n,
+        query: this.facetState.facetSearch.query,
+        numberOfMatches: this.facetState.facetSearch.values.length,
+        hasMoreMatches: this.facetState.facetSearch.moreValuesAvailable,
+      },
+    });
+  }
+
+  private get activeValues() {
+    return this.facetState.values.filter(({state}) => state !== 'idle');
+  }
+
+  private get displayName() {
+    return this.facetState.displayName || 'no-label';
+  }
+
+  private get facetValueProps(): Pick<
+    FacetValueProps,
+    | 'facetSearchQuery'
+    | 'enableExclusion'
+    | 'field'
+    | 'i18n'
+    | 'displayValuesAs'
+  > {
+    return {
+      facetSearchQuery: this.facetState.facetSearch.query,
+      displayValuesAs: 'checkbox',
+      enableExclusion: false,
```

Comment:
This makes me realize we (and that's a broad 'we', not necessarily just the DXUI team) should think of a way to support single select / exlucsion in Atomic commerce facets.

This could be new settings in the API config, or we could think of a way to do this directly in Atomic. TBD.

We could create a spike for that and add a TODO here.

Reply:
I log it for investigation